.gitignore

```diff
@@ -29,16 +29,6 @@ Pods/
 ## android
 solutions/android/obj
 
-## marmalade
```

Comment:
I need to put that back. We don't really have it in a working condition, but we keep "supporting it" by having legacy code in there